### PR TITLE
Add quick-start-user-guide and testing guide

### PR DESCRIPTION
**Pull request by devyanikota**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
-# glusterapi-python
+# python-gluster-mgmt-client
 
 Python bindings for gluster management APIs
```
Comment (phlogistonjohn) [inline README.md]: I think the introduction should note that this is specifically  target at GlusterD2 (the "up & coming" management service for gluster).

File: doc/development-guide.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Development Guide
```
Comment (phlogistonjohn) [inline doc/development-guide.md]: I would just leave out this file if it has no content.

File: doc/testing.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Testing
+
+## Unit Tests
+
+Write tests in files whose name ends with `_test.py` which ensures that they
+are picked up by `py.test` command. These test files will contain functions
+with function name matching `testXxx` pattern. Put these test files in the
+same package as the one being tested. This ensures that unexported functions
+can also be unit tested.
```
Comment (phlogistonjohn) [inline doc/testing.md]: Should this paragraph mention that tests will go in the test directory?

File: doc/development-guide.md (hunk 2 above)
Comment (sac) [reply doc/development-guide.md]: Or put a TODO line saying we will write this in a while, or anyone interested in contributing to doc can write it up.

File: doc/quick-start-user-guide.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Quick Start User Guide
+
+This guide demonstrates how to consume the python bindings for GlusterD-2.0 APIs.
+We create a two-node GlusterFS cluster using these bindings.
+
+## Setup
+This guide takes the following as an example of IPs for the two nodes:
+ * **Node 1**: `192.168.56.101`
+ * **Node 2**: `192.168.56.102`
+
```
Comment (sac) [inline doc/quick-start-user-guide.md]: I think you have to make this more comprehensive. You specify two nodes here but in the example you have a single node. It is pretty confusing.

File: README.md (hunk 1 above)
Comment (devyanikota) [reply README.md]: done.

File: doc/quick-start-user-guide.md (hunk 4 above)
Comment (devyanikota) [reply doc/quick-start-user-guide.md]: done.

File: doc/testing.md (hunk 3 above)
Comment (devyanikota) [reply doc/testing.md]: done.

**Pull request merged**